modules/045-snapshot-controller/templates/snapshot-validation-webhook/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: d8-snapshot-validation-webhook
+  namespace: kube-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "d8-snapshot-validation-webhook" "workload-resource-policy.deckhouse.io" "master")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: d8-snapshot-validation-webhook
+  updatePolicy:
+    updateMode: "Auto"
+{{- end }}
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  annotations:
+    helm.sh/hook: post-upgrade, post-install
+    helm.sh/hook-delete-policy: before-hook-creation
+  name: d8-snapshot-validation-webhook
+  namespace: kube-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "d8-snapshot-validation-webhook" )) | nindent 2 }}
+spec:
+  minAvailable: {{ include "helm_lib_is_ha_to_value" (list . 1 0) }}
+  selector:
+    matchLabels:
+      app: d8-snapshot-validation-webhook
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: d8-snapshot-validation-webhook
+  namespace: kube-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "d8-snapshot-validation-webhook" )) | nindent 2 }}
+spec:
+  replicas: {{ include "helm_lib_is_ha_to_value" (list . 2 1) }}
+  selector:
+    matchLabels:
+      app: d8-snapshot-validation-webhook
+  template:
+    metadata:
+      labels:
+        app: d8-snapshot-validation-webhook
+    spec:
+      imagePullSecrets:
+        - name: deckhouse-registry
+      {{- include "helm_lib_priority_class" (tuple . "cluster-low") | nindent 6 }}
```

Comment:
Everything which affects API interactions should have the cluster high priority class?

Reply:
priority class switched to `system-cluster-critical`